src/GeositeFramework/js/App.js

```diff
@@ -100,6 +104,32 @@
             });
         });
     };
+
+    function registerLaunchpadHandlers(launchpadConfigs) {
```

Comment:
Consider creating and moving this code to a new Launcpad module/controller (similar to SubRegion) since the feature will likely add a bit of code.  Would be nice not to have it directly embedded in App.js.